### PR TITLE
Eurostat employed per sector

### PR DESCRIPTION
**Pull request by eftekhari-mhs**

### CODE REVIEW

File: scripts/eurostat/regional_statistics_by_nuts/employed_per_sector/emp_persec_preprocess_gen_tmcf.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+import pandas as pd
+import io
+import csv
+
+_DATA_URL = "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing?file=data/nama_10r_3empers.tsv.gz"
+_CLEANED_CSV = "./Eurostats_NUTS3_Empers.csv"
+_TMCF = "./Eurostats_NUTS3_Empers.tmcf"
+
+_OUTPUT_CLOUMNS = [
```
Comment (jeffreyoldham) [inline scripts/eurostat/regional_statistics_by_nuts/employed_per_sector/emp_persec_preprocess_gen_tmcf.py:23]:
```suggestion
_OUTPUT_COLUMNS = [
```

Be also update all other instances.

**Review by jeffreyoldham — COMMENTED** [review]
Thanks.

File: scripts/eurostat/regional_statistics_by_nuts/employed_per_sector/Eurostats_NUTS3_Empers.mcf [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Person
+employment: dcs:Employed
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+sourceScalingFactor: 1000
+
+
+# Population employed by NACE code (A)
+Node: dcid:Count_Person_Employed_NACE/A
```
Comment (jeffreyoldham) [inline scripts/eurostat/regional_statistics_by_nuts/employed_per_sector/Eurostats_NUTS3_Empers.mcf:12]: I am still admittedly confused about schema.  Does this need to be split into a separate pull request containing only schema changes?

**Review by jeffreyoldham — COMMENTED** [review]
One more question.

File: scripts/eurostat/regional_statistics_by_nuts/employed_per_sector/Eurostats_NUTS3_Empers.mcf (hunk 2 above)
[the remark this answers is not in the document]
Comment (eftekhari-mhs) [reply scripts/eurostat/regional_statistics_by_nuts/employed_per_sector/Eurostats_NUTS3_Empers.mcf:12]: Yes, I checked this with Tiffany and enums are already submitted under schema.

File: scripts/eurostat/regional_statistics_by_nuts/employed_per_sector/README.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+### Download URL
+
+[TSV] file is available for [download](https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing?file=data/nama_10r_3empers.tsv.gz).
+
+[Data explorer](https://appsso.eurostat.ec.europa.eu/nui/show.do?dataset=nama_10r_3empers&lang=en)
+
+### Overview
+
+This dataset has the the employee per sector (thousand persons) for European Union (EU) countries to their NUTS3 geos. It has the employees and employed persons counts categorized by NACE classification for 15 different economic activities in [1995-2018]. The NACE categories are as follows:
```
[the remark this answers is not in the document]
Comment (eftekhari-mhs) [reply scripts/eurostat/regional_statistics_by_nuts/employed_per_sector/README.md:20]: I meant the current data has the years [1995-2018].

File: scripts/eurostat/regional_statistics_by_nuts/employed_per_sector/Eurostats_NUTS3_Empers.mcf (hunk 2 above)
Comment (eftekhari-mhs) [reply scripts/eurostat/regional_statistics_by_nuts/employed_per_sector/Eurostats_NUTS3_Empers.mcf:12]: The enums are already submitted. 
https://github.com/datacommonsorg/data/blob/33c245c287eb6585bd12cff763786738f4b6bb78/schema/nace.mcf

**Review by jeffreyoldham — APPROVED** [review]
Thanks.
